app/furniture/marketplace/order.rb

```diff
@@ -52,5 +52,126 @@ def marketplace_name
     def price_total
       [product_total, delivery_fee, tax_total].compact.sum
     end
+
+    def square_client
+      environment = ENV.fetch("SQUARE_ENV")
+      access_token = marketplace.square_access_token
+
+      @square_client ||= Square::Client.new(access_token:, environment:)
+    end
+
+    def send_to_square_seller_dashboard(stripe_balance_transaction)
+      square_create_order_response = create_square_order
+      square_order_id = square_create_order_response.body.order[:id]
+      create_square_order_payment(square_order_id)
+
+      # TODO: What to return here?
+      true
+    end
+
+    def create_square_order
```

Comment:
Can this be a private method since it's only called from the `send_to_square_seller_dashboard` method?

Reply:
Definitely! And other methods as well. `send_to_square_seller_dashboard` was meant to be the only interface exposed to consumers.